tests/python/unittest/test_ndarray.py

```diff
@@ -291,6 +293,48 @@ def test_ndarray_legacy_load():
         assert same(data[i].asnumpy(), legacy_data[i].asnumpy())
 
 
+@with_seed()
+def test_buffer_load():
+    nrepeat = 10
+    tmp = tempfile.NamedTemporaryFile(mode='wb', dir=os.path.join(os.getcwd()))
```

Comment:
Please use https://docs.python.org/3/reference/compound_stmts.html#the-with-statement
Why are you specifying a dir? Usually python uses the system temp dir. Please don't write to CWD.

Reply:
I had that at first but it was also giving me permission issues, I have pushed a change without the directory specified.